cmd/theatre-envconsul/main.go

```diff
@@ -133,6 +134,37 @@ func mainError(ctx context.Context, command string) (err error) {
 			}
 		}
 
+		var filePaths = environment{}
+		const pathTBD = ""
+
+		// Rewrite vault-file: envs to vault: envs. Store the paths to which they should
+		// be written to in filePaths
+		for key, value := range env {
+			if strings.HasPrefix(value, "vault-file:") {
+				// determine if we define a path at which to place the file
+				trimmed := strings.TrimSpace(
+					strings.TrimPrefix(value, "vault-file:"),
+				)
+				if len(trimmed) == 0 {
+					return errors.Wrap(
+						fmt.Errorf("empty vault-file env var: %v", value),
+						"failed parse vault-file env var")
```

Comment:
The wrapping is confusing and doesn't really give us anything, so I'd suggest:

```suggestion
					return fmt.Errorf("empty vault-file env var: %v", value)
```